zerver/openapi/zulip.yaml

```diff
@@ -28000,22 +28006,36 @@ components:
         - allow_empty_topic
         - disable_empty_topic
         - empty_topic_only
+      example: inherit
       description: |
-        Setting defining the policy for sending messages to the empty topic in
-        this channel.
+        Whether [named topics](/help/introduction-to-topics) and the empty
+        topic (i.e., ["general chat" topic](/help/general-chat-topic))
+        are enabled in this channel.
 
-        - inherit - Channel will inherit the organization-level setting, `realm_topics_policy`.
-        - allow_empty_topic - Topics are not required to send messages in the channel.
-        - disable_empty_topic - Topics are required to send messages in the channel.
-        - empty_topic_only - Only the empty topic is available in this channel.
+        - `"inherit"`: Messages can be sent to named topics in this channel,
+          and the [organization-level `realm_topics_policy`][realm-topics-policy]
+          is used for whether messages can be sent to the empty topic in this
+          channel.
+        - `"allow_empty_topic"`: Messages can be sent to both named topics and
+          the empty topic in this channel.
+        - `"disable_empty_topic"`: Messages can be sent to named topics in this
+          channel, but the empty topic is disabled.
+        - `"empty_topic_only"`: Messages can be sent to the empty topic in this
+          channel, but named topics are disabled. See ["general chat"
+          channels](/help/general-chat-channels).
+
+        The `"empty_topic_only"` policy can only be set if all existing messages
+        in the channel are already in the empty topic.
 
-        The `empty_topic_only` policy can only be enabled in a channel if all
-        messages in the channel are already in the empty topic.
+        When creating a new channel, if the `topics_policy` is not specified, the
+        `"inherit"` option will be set.
```

Comment:
This seemed important to note for the endpoint that creates channels where this schema is being used as a parameter. And the information didn't seem harmful to have in the other places the schema is used.

If we don't want this though, we'll have to switch from using a shared schema for this and have some duplicated text.